src/Export/EntityManager.php

```diff
@@ -326,7 +326,8 @@ public function queueExportLocalEntityAllSyncs(
       //    notes    : There are legitimate cases where we still want to allow
       //               exports of new imported entities e.g. import from one
       //               remote resource and send to another.
-      if ($is_managed && !$is_update) {
+      $sync_supports_imports = $sync->get('operations.import_list.status') || $sync->get('operations.import_list.status');
```

Comment:
@shabananavas 

Surely you meant one of the two to be `operations.import_entity.status`?

Reply:
Oops, yeah.